### ISSUE TITLE
Project is dead

### ISSUE THREAD

**Issue by MartinX3** [dmfs/opentasks#1057]
Please use https://github.com/tasks/tasks

**Comment by tieugene** [comment]
> Please use https://github.com/tasks/tasks

It wants money for CalDAV

**Comment by MartinX3** [comment]
> > Please use [tasks/tasks](https://github.com/tasks/tasks)
> 
> It wants money for CalDAV

It's GPLv3 code.
Just compile it yourself without this restriction or use the google services integration.
Of course you could also revive this dead project.
I only gave an advise for a good replacement.

But keep in mind that the development of this app costs money.

**Comment by Mte90** [comment]
It seems that on F-Droid it is not require any payment (I am using it)

**Comment by PanderMusubi** [comment]
Too bad this project is dead, really liked the app. Hope it gets revived. Also switched to tasks.org now and works fine.

**Comment by Mertica** [comment]
It's unfortunate that this project died, but at least this makes it a bit easier to choose between OpenTasks and Tasks.org

I guess I'll go with Tasks.org then

**Comment by mrtumnus** [comment]
Unfortunately, some apps rely on this project for task integration (aCalendar+).  Hopefully they will get the message and implement support for Tasks.org.

**Comment by eylenburg** [comment]
Yes, I've got the same problem. My mail provider (Infomaniak) will only sync with OpenTasks, not with Tasks.org. So I'll keep using the dead app...

**Comment by mattlaabs** [comment]
> Unfortunately, some apps rely on this project for task integration (aCalendar+). Hopefully they will get the message and implement support for Tasks.org.

We have tasks.org on our aCalendar roadmap for 2023.

**Comment by TheOtherDoctor** [comment]
I always liked the UI of OpenTasks better and thus used it actively until now. Today I paid the price, taking me some hours debugging my system as I suspected my backend being faulty unless I realized that I am also riding the dead horse, learning painfully that the OpenTasks/Task.Org choice became crucial sometime lately:
The new point I want to bring is that DAVx5 now seems to effectively support only _one_ todo list app - there is a new setup section for this, which did not grab my attention when it was introduced: In the list of candidate apps, one can activate switches for three different apps there (OpenTasks, mentioned as dead, Tasks.Org, mentioned as partially supported, and jtx Board, the new kid on the block?), but there is also a radio button on the left of the list which actually activates _one_ chosen - having that set on Tasks.Org killed my day today. Only the one app active there really gets synced against the CALDAV backend, and the icon you see at the very bottom of the DAVx5 settings list indicates which one. Huh, that took me way too much time to find out, struggling with off-synced lists from three clients.

Putting this comment here in the hope that it saves somebody the hours I wasted finding this out. So now I am also forcedly moved to tasks.org, fine.

**Comment by nosp7m** [comment]
Just in case, I created a new fork, see https://github.com/dmfs/opentasks/issues/1072 and https://github.com/nosp7m/tasksandlists.
